### DISCUSSION TITLE
Monitoring after expiration

### DISCUSSION THREAD

**Discussion by Berisch** [btcpayserver/btcpayserver#6146] [Technical Support]
Hi everyone,

I've searched through the official website, FAQs, and various third-party resources, but I couldn't find the information I'm looking for.

Could someone please guide me on how to modify the time period for which BTCPay Server monitors an invoice after its expiration? Specifically, I'm interested in the setting that controls when the message 'Invoice XXXX is not monitored anymore' is triggered.

Thanks in advance for your help!

**Comment by pavlenex** [comment]
I believe it's this one @dennisreimann @Kukks can confirm
Click on Bitcoin in the sidebar, click on the settings icon

<img width="1192" alt="Screenshot 2024-08-13 at 13 02 42" src="https://github.com/user-attachments/assets/25548412-ca7e-44ff-b6df-059b6643832a">

**Reply by dennisreimann** [reply]
It's the one in Store Settings > Payment: "Invoice expires if the full amount has not been paid after …"

![grafik](https://github.com/user-attachments/assets/4222c5cb-3776-46b6-920b-01822ec15759)

The one that @pavlenex mentioned is is addition to the expiry. See [details here](https://docs.btcpayserver.org/FAQ/Stores/#payment-invalid-if-transactions-fails-to-confirm-minutes-after-invoice-expiration).

**Comment by Berisch** [comment]
Hey @pavlenex, @dennisreimann,

Thank you for the reply!

Unfortunately, this is not the setting I was looking for (or at least it's not working for me as expected).

We have two stores on our BTCPay server. In one store, invoices are no longer monitored 7 days after expiration; in the other store, they are no longer monitored 24 hours after expiration.

Screenshot from the first store:
![image](https://github.com/user-attachments/assets/1de35479-82c4-4207-9136-0a12e588a354)
Screenshot from the second store:
![image](https://github.com/user-attachments/assets/b7433be8-846f-49d3-b6d0-7e1753c0ada0)

The setting @pavlenex mentioned was set to 1440 minutes (24 hours) for the first store, and lower (unfortunately, I don't remember the exact number) for the second store. However, even after setting it to 1440 minutes for the second store, it still stops monitoring after one day.

I would be grateful for further guidance!
